Include/abstract.h

```diff
@@ -14,9 +14,9 @@ extern "C" {
 
    Print an object 'o' on file 'fp'.  Returns -1 on error. The flags argument
    is used to enable certain printing options. The only option currently
-   supported is Py_Print_RAW.
+   supported is Py_PRINT_RAW.
 
-   (What should be said about Py_Print_RAW?). */
+   (What should be said about Py_PRINT_RAW?). */
```

Comment:
If you modify this comment, please answer the question. By default (flags=0), PyObject_Print() calls PyObject_Repr(), whereas Py_PRINT_RAW calls PyObject_Str(). See the doc:
https://docs.python.org/dev/c-api/object.html#c.PyObject_Print